cfgen/cfgen

```diff
@@ -848,44 +848,58 @@ def oids_to_dhall(oids: List[Oid]) -> str:
 NetProtocol = Enum('NetProtocol', 'tcp o2ib')
 
 
-ProcT = Enum('ProcT', 'hax m0_server m0_client_s3 m0_client_other')
+ProcT = Enum('ProcT', 'hax m0_ios m0_confd m0_client_s3 m0_client_other')
```

Comment:
I was thinking of avoiding further classification of m0_server. Can we use m0ProcT instead?

Reply:
Done